### DISCUSSION TITLE
@Mashb1t can you please contact me?

### DISCUSSION THREAD

**Discussion by wrecksone** [lllyasviel/Fooocus#2624] [General]
Mashb1t I am looking to potentially hire you if you are free. Can you please contact me at plattyai@proton.me. Thank you very much!!

**Comment by mashb1t** [comment]
Probably not, but feel free to provide more details using LinkedIn (see in my profile).

**Reply by wrecksone** [reply]
I have tried every other avenue that is why I am messaging you on here as a last resort. I cannot message you on Linkedin because we are not connections. I sent you a connection request from Michael W..... I have also tried your email but the link from your website is broken. If you accept the request then I will message you on there. If you don't like my proposition then you can remove me and go about your day no harm no foul. Thank you very much for your time.
